sql/walrus--0.1.sql

```diff
@@ -5,7 +5,6 @@
 
 create schema realtime;
 grant usage on schema realtime to postgres;
```

Comment:
@olirice we can take this out too since `realtime` schema is created by `postgres`

Reply:
fixed in 96bb612